packages/features/lend/src/lend-variable/components/lend-variable-trade-summary.tsx

```diff
@@ -0,0 +1,90 @@
+import { useContext } from 'react';
+import { Box, useTheme } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import {
+  // DataTable,
+  Faq,
+  FaqHeader,
+  TotalBox,
+  // InteractiveAreaChart,
+} from '@notional-finance/mui';
+import { useLendVariableFaq } from '../hooks';
+import { LendVariableContext } from '../../lend-variable/lend-variable';
+import {
+  TradeActionSummary,
+  useVariableTotals,
+  // useMaturitySelect,
+  // useInteractiveMaturityChart,
+  // useFixedLiquidityPoolsTable,
+} from '@notional-finance/trade';
+
+export const LendVariableTradeSummary = () => {
+  const theme = useTheme();
+  const context = useContext(LendVariableContext);
+  const { state } = context;
+  const { faqs, faqHeaderLinks } = useLendVariableFaq(
+    state.selectedDepositToken
+  );
+  const totalsData = useVariableTotals(state);
+  // const { selectedDepositToken, deposit } = state;
```

Comment:
Remove all the commented out code, we can add back what we actually end up using later